eventmesh-admin/README.md

```diff
@@ -4,10 +4,10 @@ EventMesh Administration Module for EventMesh. It manages Admin Service, Configu
 
 ## Administration Client Manager APIs
 
-### POST /clientmanage/topics/
+### POST /clientmanage/topics
```

Comment:
@yzhao244 I suggest /topicmanage instead of /clientmanage/topics here, what do you think?